kommunicateui/src/main/res/values/strings.xml

```diff
@@ -1,48 +1,48 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
 
```

Comment:
fir the changes here, i went through the google UI guidelines and referred to the examples and how they have used punctuation and made the changes. Titles don't have full stops, same with interactive text, buttons, clickable etc.
and information containing text is usually well punctuated.